src/linux/mod.rs

```diff
@@ -156,7 +156,6 @@ impl<'a> SharedLibraryTrait for SharedLibrary<'a> {
 
     #[inline]
     fn virtual_memory_bias(&self) -> Bias {
-        assert!((self.addr as usize) < (isize::MAX as usize));
```

Comment:
I'm not sure when the shared object can be offset in a file it was loaded from. If it can be offset, then indeed a signed type that's twice as big as `usize` should be used here. If it cannot, then replacing `isize` with `usize` in the `Bias` should be sufficient.

Reply:
Actually considering the size of `dlpi_addr` itself, perhaps making `Bias` a `usize` would be the right solution?